pkgs/by-name/sg/sglauncher/package.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, fetchgit, pkgs, }:
+
+stdenv.mkDerivation {
+  pname = "sglauncher";
+  version = "1.0";
+  src = fetchgit {
+    url = "https://codeberg.org/ItsZariep/SGLauncher";
+    hash = "sha256-NEAEqH4r7GG8f41ysbmGEyiWPYrc792rTlmMBp2csmw=";
+  };
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
  pname = "sg-launcher";
  version = "1.1";

  src = fetchFromGitea {
    domain = "codeberg.org";
    owner = "ItsZariep";
    repo = "SGLauncher";
    rev = finalAttrs.version;
    hash = "sha256-zIhutvtNA0UJ8+nMWv2t6hMpO9wkYI2sjSNT78ggn58=";
  };
```
1. Why don't use the latest version (1.1)?
2. Replace `fetchgit` with `fetchFromGitea`
3. Use `finalAttrs` pattern: `stdenv.mkDerivation (finalAttrs: { ... })`